src/main/java/com/amazon/ion/impl/Ion11Writer.kt

```diff
@@ -0,0 +1,153 @@
+package com.amazon.ion.impl
+
+import com.amazon.ion.Decimal
+import com.amazon.ion.IonType
+import com.amazon.ion.Timestamp
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.time.Instant
+import java.time.LocalDate
+
+/**
+ * Writes Ion 1.1 data to an output source.
+ *
+ * This interface allows the user to write Ion data without being concerned about which output format is being used.
+ */
+interface Ion11Writer {
```

Comment:
We can change this if necessary, but we should decide on a convention for naming Ion 1.1 classes/methods and stick with it. I think we've both used `_1_1` in some places, which is ugly due to the underscores but at least doesn't look like the number eleven. If we go with `11` as a convention, should it be `Ion11Writer` or `IonWriter11`?

Reply:
I kind of like `Ion11...`, but I'll go with `IonWriter_1_1` for now to be consistent with what we already have.